scripts/firedrake-install

```diff
@@ -1425,7 +1425,7 @@ def build_and_install_slepc():
 
 def build_and_install_netgen():
     log.info("Installing NGSolve/Netgen.")
-    run_pip(["install", "netgen-mesher==6.2.2304.post142.dev0"])
```

Comment:
Why do you want to unpin the version of netgen rather than pinning to a new stable version that we know works with Firedrake ?

Reply:
We normally do not pin unless there is an issue. Do we have a reason to pin?

Reply:
I do not think we need to pin now. If we see some issue in the future, we might need to pin, but that is not a big deal.